src/util/NuulsUploader.cpp

```diff
@@ -43,6 +43,7 @@ void uploadImageToNuuls(RawImageData imageData, ChannelPtr channel,
     const static QString contentType =
         QString("multipart/form-data; boundary=%1").arg(boundary);
     static QUrl url(Env::get().imageUploaderUrl);
+    static QString FormBody(Env::get().imageUploaderFormBody);
```

Comment:
we use `camelCase` for local variables, not `PascalCase`

Reply:
gotcha, updated the variable name